site/en/codelabs/telink-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,739 @@
+
+### Linux Machine
+
+The Linux machine (Ubuntu v20.04 LTS or later) acts as the build machine to set up the Telink Zephyr development environment and flash all Thread development boards.
+To accomplish these tasks, the Linux machine requires two available USB ports and internet connectivity.
+
+### Serial Port Connection and Terminals
+
+It is recommended to be familiar with establishing connections between devices using a serial port converter cable.
+Additionally, you will need a serial port terminal tool to access the devices.
+
+In this Codelab, terminal tool PuTTY is used for controlling the FTD Joiner and Raspberry Pi.
+It offers an overview of its usage, but other terminal software can also be employed.
+
+### Telink TLSR9518 Development Kit
+
+This Codelab needs two TLSR9518 development kits.
+
+<img src="img/overview.jpg" alt="overview.jpg" width="624.00" />
+
+One of these kits will be used as RCP (Radio Co-Processor), while the other will function as FTD (Full Thread Device).
+If you are not yet in possession of this board, you can acquire more details of TLSR9518 development kit from the [Telink official website](http://wiki.telink-semi.cn/wiki/Hardware/B91_Generic_Starter_Kit_Hardware_Guide/).
+Some components to be used are as follows:
+
+| Index | Name                                         |
+| :---- | -------------------------------------------- |
+| 1     | Telink TLSR9518 Development Kit              |
+| 2     | Telink Burning Kit                           |
+| 3     | 2.4Ghz Antenna                               |
+| 4&5   | USB Cable (USB A to mini USB)                |
+| 6     | USB to TTL Serial Cable                      |
+
+> **Note**: Item 6 is not included in the kit for sale.
+
+### Raspberry Pi 3B+ or greater with Raspbian OS Image
+
+In this Codelab, a Raspberry Pi 3B+ or greater with [Raspbian Bullseye Lite OS image](https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf-lite.img.xz) or [Raspbian Bullseye with Desktop](https://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf.img.xz) is needed.
+It is connected to the internet via Ethernet and will be configured as the host for the OpenThread Border Router (OTBR).
+
+### Network Connection
+
+An internet-connected switch (or router) and several Ethernet cables.
+They are used to connect the Raspberry Pi to the Linux machine, facilitating user configuration of the Raspberry Pi via the host.
+
+### LinuxBDT
+
+Telink [Burning and Debugging Tool (BDT)](http://wiki.telink-semi.cn/wiki/IDE-and-Tools/Burning-and-Debugging-Tools-for-all-Series/), which applies to all Telink Chip series, allows you to erase and flash OpenThread firmware onto the Telink TLSR9518 Development Kits.
+Please install the X86-based linux version [linuxBDT](http://wiki.telink-semi.cn/tools_and_sdk/Tools/BDT/LinuxBDT.tar.bz2) on your Linux machine.
```

Comment:
We don't use "please" in our documents per the Google style guide https://github.com/openthread/ot-docs/blob/main/STYLE_GUIDE.md#style

Can you remove it here and anywhere else it's used?  Should be able to just take it out without having to change the rest of the sentence.

Reply:
Words have been adjusted in the updated commit, removing 'please.'